base/strings/substring.jl

```diff
@@ -136,18 +136,23 @@ reverse-order iteration without making a copy. Custom string types must implemen
 and encoding. If they return a string with a different encoding, they must also override
 `reverseind` for that string type to satisfy `s[reverseind(s,i)] == reverse(s)[i]`.
 
+!!! note
+    The examples below may be rendered differently on different systems.
+    The comments indicate how they're supposed to be rendered.
+
 # Examples
 ```jldoctest
 julia> reverse("JuliaLang")
 "gnaLailuJ"
 
-julia> reverse("ax̂e") # combining characters can lead to surprising results
-"êxa"
+# combining characters can lead to surprising results
```

Comment:
Probably better to just start a new `jldoctest` block and put this comment and the `!!! note` before the start.